frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -41,6 +48,8 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
   const deviceSets: DeviceSet[] = ocsConfig?.spec.storageDeviceSets || [];
 
   const [response, loadError, loading] = usePrometheusPoll(cephCapacityResource);
+  const [pvData] = useK8sWatchResource<K8sResourceKind[]>(pvResource);
+  const [nodesData] = useK8sWatchResource<NodeKind[]>(nodeResource);
```

Comment:
dataLoaded and dataError checks might not be required here... in case of such errors `pvData === [] & nodesData === []` and `isArbiterSC & isNormalSC` functions can handle that...
I do not think there will be any use case where dataLoaded and dataError fields are required here ??